index.html

```diff
@@ -64,5 +64,214 @@ <h1 class="header__connect">
+          <a href="#" class="see__project">See Project</a>
+        </button>
+      </div>
+
+      <div class="works__cards">
+        <img src="./images/card-2.svg" alt="" />
+        <h1 class="project__title">Multi-Post Stories</h1>
+        <ul class="client">
+          <li class="project__name">Canopy</li>
+          <li class="counter">
+            <img src="./images/Counter.svg" alt="" />
+          </li>
+          <li class="project__role">Back End Dev</li>
+          <li class="counter">
+            <img src="./images/Counter.svg" alt="" />
+          </li>
+          <li class="year">2015</li>
+        </ul>
+        <p class="project__description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="html">
+            <a href="#" class="language">html</a>
+          </li>
+          <li class="css">
+            <a href="#" class="language">css</a>
+          </li>
+          <li class="javascript">
+            <a href="#" class="language">JavaScript</a>
+          </li>
+        </ul>
+        <button type="button" class="btn">
+          <a href="#" class="see__project">See Project</a>
+        </button>
+      </div>
+
+      <div class="works__cards">
+        <img src="./images/card-3.svg" alt="" />
+        <h1 class="project__title">Tonic</h1>
+        <ul class="client">
+          <li class="project__name">Canopy</li>
+          <li class="counter">
+            <img src="./images/Counter.svg" alt="" />
+          </li>
+          <li class="project__role">Back End Dev</li>
+          <li class="counter">
+            <img src="./images/Counter.svg" alt="" />
+          </li>
+          <li class="year">2015</li>
+        </ul>
+        <p class="project__description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="html">
+            <a href="#" class="language">html</a>
+          </li>
+          <li class="css">
+            <a href="#" class="language">css</a>
+          </li>
+          <li class="javascript">
+            <a href="#" class="language">JavaScript</a>
+          </li>
+        </ul>
+        <button type="button" class="btn">
+          <a href="#" class="see__project">See Project</a>
+        </button>
+      </div>
+    </section>
+
+    <!-- /**
+        *! About Myself Section Here.
+        **/ -->
+
+    <section class="about__myself__container">
+      <div class="about__self">
+        <h1 class="about__myself__title">About<br> Myself</h1>
+        <p class="about__myself__description">
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don't hestiate to
+          contact me.
```

Comment:
- [ ] Kindly consider applying the same layout change I suggested in the works section. Our design in screens bigger than `768px` must be similar to what we have in the figma design. 
- [ ] Also there is `margin: 8px` applied to `body` from maybe `scss` stylesheets that is kind of unnecessary and it makes your design less similar to the figma design. kindly consider removing that.
your design
![vc](https://user-images.githubusercontent.com/96683843/219740440-5e1749ba-628d-4bfb-896a-6991dcd1f7f2.jpg)

figma design
![zx](https://user-images.githubusercontent.com/96683843/219740401-685119c8-be81-4334-972a-0f12beb3240e.jpg)

Reply:
Dear @Zuhaib042, thank you very much for your feedback and suggestion. however, your requested change is not part of this project.  It will only be applicable when we add media queries and cards for the "desktop version"

Reply:
@tsheporamantso I am really sorry about that. that was actually a misunderstanding on my part. I really apologize for my shortcoming.